doc/images/blockdiagram.svg

```diff
@@ -466,15 +466,15 @@
        y="243.02469"
        id="text959-9"><tspan
          sodipodi:role="line"
-         id="tspan957-2"
          x="56.885418"
          y="243.02469"
-         style="stroke-width:0.26458332px">IF</tspan><tspan
+         style="stroke-width:0.26458332px"
+         id="tspan1288">ID</tspan><tspan
          sodipodi:role="line"
          x="56.885418"
          y="247.98563"
          style="stroke-width:0.26458332px"
-         id="tspan1288">IDE</tspan></text>
+         id="tspan1260">EX</tspan></text>
```

Comment:
I think it should rather be IF-ID/EX because it separates IF from ID/EX but it's maybe too long?

Reply:
I did have a go at doing this, but I didn't think that it fitted well within the diagram so left it as is.  The register is feeding the ID/EX stage so I think it's reasonable to leave it named after it.

Reply:
Yes, I agree.